target/linux/pistachio/base-files/etc/init.d/bt-csr

```diff
@@ -0,0 +1,191 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright 2016 by Imagination Technologies Limited and/or its affiliated group companies.
```

Comment:
Well this is not BSD-3C license as per https://imghub.imgtec.org/display/IM/Ci40+Top+level?preview=%2F71337119%2F72554420%2FCreator+Ci40+3-clause+BSD+license+1Mar2016+RL.docx however in openwrt they need standard OpenWrt license header https://github.com/CreatorDev/openwrt/blob/ci40/target/linux/pistachio/marduk/target.mk